test/hooks/index.js

```diff
@@ -619,6 +619,42 @@ module.exports = input;
 			});
 	});
 
+	it('supports renderChunk in place of transformBundle and transformChunk', () => {
```

Comment:
To have comprehensive tests for the only Chunk transform hook we want to keep (as of today 😉), I think we should extend this test to check that without returning anything, the output remains unchanged (this is the "get information about the Chunk" part of the hook) and add a second test that only tests that this hook actually transforms the chunk.

Reply:
For a long term perspective, I think that actually all `transformBundle` tests should be reflected in corresponding `renderChunk` tests as otherwise the new hooks are not nearly as well tested as the old hooks while it should actually be the other way around. Once we reach hard deprecations for the old hooks, quite a few tests can be removed again.

Reply:
Agreed, will do.